services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -110,12 +110,28 @@ export const DetailSection = ({
           )}
         </section>
         <section className="detail-section ds-u-margin-bottom--7">
-          <FileList
-            heading={pageConfig.attachmentsHeading}
-            infoText={downloadInfoText}
-            uploadList={detail.attachments}
-            zipId={detail.componentId}
-          />
+          {detail.attachments.length > 0 ? (
+            <FileList
+              heading={pageConfig.attachmentsHeading}
+              infoText={downloadInfoText}
+              uploadList={detail.attachments}
+              zipId={detail.componentId}
+            />
+          ) : (
+            <>
+              <h2>{pageConfig.attachmentsHeading}</h2>
+              <Review
+                className="original-review-component preserve-spacing"
+                headingLevel="2"
+              >
+                <i>
+                  At this time, the attachments for this package are unavailable
+                  in this system. Contact your CPOC to verify the initial
+                  submission documents.
+                </i>
+              </Review>
+            </>
+          )}
```

Comment:
This works. It would be nice though if the missing file message were integrated into the FileList component...maybe as a passed in customized message. Then we wouldnt have to duplicate the h2 header either.